addon/initializers/route-layers.js

```diff
@@ -10,10 +12,10 @@ export function initialize (container, application) {
     routeLayer: 'default',
 
     afterModel: function (model, transition) {
-      this._super.apply(this, arguments);
+      this._super(...arguments);
 
       // Leaf route only
-      var leafRouteName = transition.handlerInfos.get('lastObject.handler.routeName');
+      var leafRouteName = Ember.A(transition.handlerInfos).get('lastObject.handler.routeName');
```

Comment:
Since we're relying on `.get` being present, we need to ensure that `handlerInfos` is an Ember Array.